server/py/services/api/api/endpoints/model_endpoints.py

```diff
@@ -239,14 +239,15 @@ async def list_model_endpoints(
     List model endpoints.
 
     :param project:         The name of the project.
-    :param names:            The model endpoints names.
+    :param names:           The model endpoints names.
     :param model_name:      The model name.
     :param function_name:   The function name.
     :param function_tag:    The function tag.
     :param labels:          The labels of the model endpoint.
     :param start:           The start time to filter by.Corresponding to the `created` field.
     :param end:             The end time to filter by. Corresponding to the `created` field.
-    :param tsdb_metrics:    Whether to include metrics from the time series DB.
+    :param tsdb_metrics:    Comma separated list of metric names to add. Defaults to all available metrics. To disable,
+                            set to "false".
```

Comment:
This is confusing logic, though I don't see a good way to have consistent logic with a single parameter.
I think it's better to be verbose here and separate to 2 params:
* `with-tsdb-metrics` - default to `True`.
* `tsdb-metrics` - if existing, this is the list of metrics. If `None` (the default) should behave according to the `with-tdsb-metrics` value, meaning either all metrics or no metrics.

Reply:
@theSaarco do we want to do some of this changes in 1.8.0 in order to avoid BC issues?